client/src/components/polls/PollModal.js

```diff
@@ -1,16 +1,34 @@
-import React from 'react';
+import React, { useCallback, useEffect, useContext } from 'react';
 import axios from 'axios';
 import { makeStyles } from '@material-ui/core/styles';
 import { Button, Modal, TextField } from '@material-ui/core';
+import { useDropzone } from 'react-dropzone';
+import { UserContext } from '../UserContext';
 
 export default function PollModal(props) {
   const classes = useStyles();
-  const [open, setOpen] = React.useState(false);
+  const [open, setOpen] = React.useState(true);
```

Comment:
It should start as false, I imagine you set to true for debugging.

Reply:
Yup. It was meant for debugging.